overlord/devicestate/devicemgr.go

```diff
@@ -765,6 +764,11 @@ type System struct {
 	Actions []SystemAction
 }
 
+var currentSystemActions = []SystemAction{
+	{Title: "recover", Mode: "recover"},
```

Comment:
I would expect this to be uppercase at least, maybe you will chat with Gustavo about these

Reply:
Capitalized? I'll add capitalization on the console-conf side too.

Reply:
yes, sorry, I meant "Capitalized"

Reply:
Also tweaked the action titles, it's `Reinstall` for current system, but `Install` for other systems.